### DISCUSSION TITLE
Can you make the control center support brightness and volume control?

### DISCUSSION THREAD

**Discussion by lemon112211** [MonitorControl/MonitorControl#993] [Ideas]
first of all, thank you for developing such a wonderful app that can help us control the monitor easily, but I am wondering if there is any available ways that could make the control center of the macOS support the brightness and volume control? if you can do it, that will be really  excellent and perfect!
I am a user from china and I am gonna thank you again for what you have done.

**Comment by waydabber** [comment]
Hi @lemon112211 - unfortunately control center is fully locked down and we can't hook into that slider. :( if you have a built-in screen and can control the built-in screen's brightness with that slider, MonitorControl is able to sync the brightness level of the internal display to the external display. So it's a kind of indirect control through that slider. But this works only for this specific use-case.
